SupportFiles/ws-tools.sh

```diff
@@ -0,0 +1,161 @@
+ln -s /opt/eclipse/eclipse /usr/local/bin/eclipse
+
+cp /etc/cfn/tools/eclipse/eclipse.desktop /usr/share/applications/eclipse.desktop
+cp /etc/cfn/tools/eclipse/eclipse.desktop /home/WorkstationUser/.local/share/applications/eclipse.desktop
+chown WorkstationUser:WorkstationUser /home/WorkstationUser/.local/share/applications/eclipse.desktop
+chmod 600 /home/WorkstationUser/.local/share/applications/eclipse.desktop
+
+#Install Intellij
+tar xfz /etc/cfn/tools/intellij/ideaIC-2018.3.3.tar.gz -C /opt/ || err_exit "Failed to unzip ideaIC-2018.3.3.tar.gz"
+chmod -R 755 /opt/idea-IC-183.5153.38
+ln -s /opt/idea-IC-183.5153.38/bin/idea.sh /usr/local/bin/idea
+
+cp /etc/cfn/tools/intellij/jetbrains-idea-ce.desktop /usr/share/applications/jetbrains-idea-ce.desktop
+cp /etc/cfn/tools/intellij/jetbrains-idea-ce.desktop /home/WorkstationUser/.local/share/applications/jetbrains-idea-ce.desktop
+chown WorkstationUser:WorkstationUser /home/WorkstationUser/.local/share/applications/jetbrains-idea-ce.desktop
+chmod 600 /home/WorkstationUser/.local/share/applications/jetbrains-idea-ce.desktop
+
+#Install emacs
+yum -y install emacs || err_exit "Failed to install emacs"
+
+#Install Gradle
+unzip -d /opt/ /etc/cfn/tools/gradle/gradle-5.1.1-bin.zip
+chmod -R 755 /opt/gradle-5.1.1
+ln -s /opt/gradle-5.1.1/bin/gradle /usr/local/bin/gradle
+
+# Install Maven
+yum -y install maven || err_exit "Failed to install maven"
+
+# Install git
+rpm -Uvh /etc/cfn/tools/git/endpoint-repo-1.7-1.x86_64.rpm
+yum -y install git  || err_exit "Failed to install git"
+yum -y install git-gui || err_exit "Failed to install git-gui"
+
+# Install ruby
+yum -y install ruby || err_exit "Failed to install ruby"
+
+# Install node.js
+tar -xvf  /etc/cfn/tools/nodejs/node-v11.6.0-linux-x64.tar.xz -C /opt
+chmod -R 755 /opt/node-v11.6.0-linux-x64
+ln -s /opt/node-v11.6.0-linux-x64/bin/node /usr/bin/node
+ln -s /opt/node-v11.6.0-linux-x64/bin/npm /usr/bin/npm
+
+# Install pycharm
+tar -xvf  /etc/cfn/tools/pycharm/pycharm-community-2018.3.3.tar.gz -C /opt
+chmod -R 755 /opt/pycharm-community-2018.3.3
+ln -s /opt/pycharm-community-2018.3.3/bin/pycharm.sh /usr/local/bin/pycharm 
+
+cp /etc/cfn/tools/pycharm/pycharm.desktop /usr/share/applications/pycharm.desktop
+cp /etc/cfn/tools/pycharm/pycharm.desktop /home/WorkstationUser/.local/share/applications/pycharm.desktop
+chown WorkstationUser:WorkstationUser /home/WorkstationUser/.local/share/applications/pycharm.desktop
+chmod 600 /home/WorkstationUser/.local/share/applications/pycharm.desktop     
+
+# Install asciidoctor tool chains 
+rpm -Uvh /etc/cfn/tools/asciidoctor/rubygem-asciidoctor-1.5.6.1-1.el7.noarch.rpm  
+yum -y install rubygem-asciidoctor || err_exit "Failed to install rubygem-asciidoctor"
+
+# Install Visual Studio Code
+rpm -Uvh /etc/cfn/tools/vscode/code-1.30.2-1546901769.el7.x86_64.rpm  
```

Comment:
See prior comment about using `rpm` command (and the IntelliJ comment's version number abstraction)

Reply:
First part - not done. Need help.
Second part - done. Checked in changes already.